x-pack/test/functional/apps/code/search.ts

```diff
@@ -36,12 +37,23 @@ export default function searchFunctonalTests({ getService, getPageObjects }: Tes
 
       it('Trigger symbols in typeahead', async () => {
         log.debug('Trigger symbols in typeahead');
-        // Fill in the search query bar with a common prefix of symbols.
         await PageObjects.code.fillSearchQuery('user');
 
+        await retry.tryForTime(5000, async () => {
+          const symbols = await testSubjects.findAll(fileTypeaheadListSelector);
+          expect(symbols).to.have.length(1);
+
+          expect(await symbols[0].getVisibleText()).to.equal('LICENSE');
+        });
+      });
+
+      it('File typeahead should be case insensitive', async () => {
+        log.debug('File typeahead should be case insensitive');
+        await PageObjects.code.fillSearchQuery('LICENSE');
```

Comment:
can you do both `LICENSE` and `license`?

Reply:
will do